app/docs/async-api.yaml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.4.0
+info:
+  title: FFC Payment Statement Constructor
+  version: '1.0.0'
+  description: Construct data for Payment Statements
+channels:
+  reurn/message:
+    subscribe:
+      message:
+        $ref: 
```

Comment:
`'#/components/messages/ReturnMessage'`

references what we define from line 13 onwards